packages/sirius-web/backend/sirius-web-papaya/src/main/java/org/eclipse/sirius/web/papaya/representations/table/PackageTableRepresentationDescriptionProvider.java

```diff
@@ -171,6 +181,14 @@ private List<ICellDescription> getCellDescriptions() {
                 .cellOptionsLabelProvider(new CellOptionLabelProvider(this.labelService))
                 .cellOptionsProvider(new CellOptionsProvider(this.composedAdapterFactory))
                 .build());
+        cellDescriptions.add(IconLabelCellDescription.newIconLabelCellDescription("iconLabelCells")
+                .canCreatePredicate(variableManager -> variableManager.get(ColumnDescription.COLUMN_TARGET_OBJECT, Object.class).filter(String.class::isInstance).map(String.class::cast)
+                        .filter(value -> value.equals("IconColumn")).isPresent())
+                .targetObjectIdProvider(new TableTargetObjectIdProvider(this.identityService))
+                .targetObjectKindProvider(new TableTargetObjectKindProvider(this.identityService))
+                .cellValueProvider((variableManager, o) -> "")
```

Comment:
`o`?

Reply:
All those BiFunction will have to be changed later to use the variable manager to give access to the row, column, cell target object and more... No need to create a special use case to give access to the column target object.